### DISCUSSION TITLE
Account Security | Educational Plan

### DISCUSSION THREAD

**Discussion by Pejayy-05** [community/community#138636] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello last month I sent a support ticket about my account being hacked but haven't received any reply since then, its been a long time, i really don't know why the support is slow.

So the thing is the account got hacked is using my school email for the educational plan for copilot, now i though when i unlink the school email (the one im using now) and create a new account with it, i can use the educational plan again, I was wrong, now im confused and don't know what to do, is it possible to get the educational plan back and use it on this new account? (the one im posting with) I really really need it, it helps me at school, (i'm a 2nd year cs student). Please any reply would help especially from mods.

**Comment by ebndev** [comment]
Hi @Pejayy-05,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
